djangomako/backends.py

```diff
@@ -11,7 +11,10 @@
 import tempfile
 
 from django import get_version
-from django.contrib.staticfiles.templatetags.staticfiles import static
+try:
+    from django.contrib.staticfiles.templatetags.staticfiles import static  # Django < 3
```

Comment:
I don't think we need this as it's no longer supported.

Reply:
@iamjazzar updated, please review again.